sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2610,7 +2634,7 @@ object GpuOverrides {
         }),
     exec[CollectLimitExec](
       "Reduce to single partition and apply limit",
-      ExecChecks(TypeSig.commonCudfTypes + TypeSig.DECIMAL, TypeSig.all),
+      ExecChecks(TypeSig.all - TypeSig.UDT.nested(), TypeSig.all),
```

Comment:
Actually thinking about it more, we might have an issue with CalendarInterval, binary, and a few other types. I'm not sure that we support them on the GPU so calling them out as supported here might end up causing us to try to move them to the GPU and get an exception.  Can we either remove them from here as listed as supported, or make sure that we have a test somewhere to verify that they work as expected?